go-controller/pkg/ovn/default_network_controller.go

```diff
@@ -587,6 +587,54 @@ func (oc *DefaultNetworkController) Run(ctx context.Context) error {
 	return nil
 }
 
+func (oc *DefaultNetworkController) Reconcile(netInfo util.NetInfo) error {
```

Comment:
clever trick making the default network controller reconcilable by network manager, even though it acts as its own controller....wont add any confusion at all 8-)

Reply:
lol I am open to suggestions, if you have one, I commit to the rework. You know how I always tell you that we should treat the default network just as any other network as much as we can? I pitched that idea since the early multi-net days. I wish I made that happen then, now it is haunting me.

Reply:
i agree with you this is the right direction